pkg/steps/artifacts.go

```diff
@@ -216,6 +217,36 @@ type PodClient interface {
 	podCmdExecutor
 }
 
+// Allow tests to accelerate time
+var timeSecond = time.Second
+
+func waitForContainer(podClient PodClient, ns, name, containerName string) error {
+	logrus.WithFields(logrus.Fields{
+		"namespace": ns,
+		"name":      name,
+		"container": containerName,
+	}).Trace("Waiting for container to be running.")
+
+	return wait.PollImmediate(time.Second, 30*timeSecond, func() (bool, error) {
```

Comment:
Could e.g. volume attach take longer than 30s? Do we care?

Reply:
Probably, the 30 seconds was a random guess. Failures here are non-terminal though, we just log them. 30 seconds is definitely better than not waiting at all. We can use ci-search to figure out if this is too little later on and if yes finetune